substrate/frame/revive/src/benchmarking.rs

```diff
@@ -471,6 +472,36 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark(pov_mode = Measured)]
+	fn eth_substrate_call(c: Linear<0, { 100 * 1024 }>) -> Result<(), BenchmarkError> {
+		let caller = whitelisted_caller();
+		T::Currency::set_balance(&caller, caller_funding::<T>());
+		let origin = Origin::EthTransaction(caller);
+		DebugSettings::default().allow_eth_substrate_call().write_to_storage::<T>();
+		let dispatchable = frame_system::Call::remark { remark: vec![42u8; c as usize] }.into();
+		#[extrinsic_call]
+		_(origin, Box::new(dispatchable));
+
+		Ok(())
+	}
```

Comment:
I don't think we need that if we did it will overlap with  with_ethereum_context_no_events